crates/api_models/src/payments.rs

```diff
@@ -4153,11 +4153,43 @@ pub struct RewardData {
     pub merchant_id: id_type::MerchantId,
 }
 
-#[derive(Debug, Clone, Eq, PartialEq, serde::Serialize, serde::Deserialize, ToSchema)]
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct BoletoVoucherData {
-    /// The shopper's social security number
+    /// The shopper's social security number (CPF or CNPJ)
     #[schema(value_type = Option<String>)]
     pub social_security_number: Option<Secret<String>>,
+
+    /// The shopper's bank account number associated with the boleto
+    #[schema(value_type = Option<String>)]
+    pub bank_number: Option<String>,
+
+    /// The type of identification document used (e.g., CPF or CNPJ)
+    #[schema(value_type = Option<DocumentKind>, example = "Cpf", default = "Cnpj")]
+    pub document_type: Option<common_enums::DocumentKind>,
+
+    /// The fine percentage charged if payment is overdue
+    #[schema(value_type = Option<f64>)]
+    pub fine_percentage: Option<FloatMajorUnit>,
+
+    /// The number of days after the due date when the fine is applied
+    #[schema(value_type = Option<i64>)]
+    pub fine_quantity_days: Option<MinorUnit>,
+
+    /// The interest percentage charged on late payments
+    #[schema(value_type = Option<f64>)]
+    pub interest_percentage: Option<FloatMajorUnit>,
+
+    /// The number of days after which the boleto is written off (canceled)
+    #[schema(value_type = Option<i64>)]
+    pub write_off_quantity_days: Option<MinorUnit>,
+
+    /// Custom messages or instructions to display on the boleto
+    #[schema(value_type = Option<Vec<String>>)]
+    pub messages: Option<Vec<String>>,
+
+    #[schema(value_type = Option<NaiveDate>)]
+    // The date upon which the boleto is due and is of format: "YYYY-MM-DD"
+    pub due_date: Option<chrono::NaiveDate>,
```

Comment:
can we directly use `time::Date` instead of `chrono::NaiveDate`?
```suggestion
    #[schema(value_type = Option<Date>)]
    // The date upon which the boleto is due and is of format: "YYYY-MM-DD"
    pub due_date: Option<time::Date>,
```

Reply:
If there isn’t a functional difference, we could stick with Date from the existing crate and avoid adding a dependency on Chrono.